cypress/e2e/adgangsplatformen.cy.ts

```diff
@@ -95,7 +95,7 @@ describe("Adgangsplatformen", () => {
     cy.get(".footer").should("not.exist");
   });
 
-  it("can register a new user - expose the right tokens for the react apps - and force logout.", () => {
+  it.skip("can register a new user - expose the right tokens for the react apps - and force logout.", () => {
```

Comment:
Add a code comment here, why we're skipping / why we're still keeping it

Reply:
Agree. As I see it this is a major problem holding back this PR.